bbot/test/test_step_2/module_tests/test_module_censys.py

```diff
@@ -1,4 +1,5 @@
 from .base import ModuleTestBase
+import json
```

Comment:
I think this import can be removed

Reply:
https://github.com/blacklanternsecurity/bbot/pull/2352/commits/35604fbbe979746f06d16dbb2f91415e57a480fc